File: java/BarangElektronik.java
public class BarangElektronik {
    private int id;
    private String nama;
    private String jenis;
    private String merek;
    private double harga;
    // Constructor
    public BarangElektronik(int id, String nama, String jenis, String merek, double harga) {
        this.id = id;
        this.nama = nama;
        this.jenis = jenis;
        this.merek = merek;
        this.harga = harga;
    }
    // Getters
    public int getId() {
        return id;
    }
    public String getNama() {
        return nama;
    }
    public String getJenis() {
        return jenis;
    }
    public String getMerek() {
        return merek;
    }
    public double getHarga() {
        return harga;
    }
    // Setters
    public void setNama(String nama) {
        this.nama = nama;
    }
    public void setJenis(String jenis) {
        this.jenis = jenis;
    }
    public void setMerek(String merek) {
        this.merek = merek;
    }
    public void setHarga(double harga) {
        this.harga = harga;
    }
}

File: cpp/BarangElektronik.cpp
#include <bits/stdc++.h>

using namespace std;

class BarangElektronik {
    private:
        int id;
        string nama;
        string jenis;
        string merek;
        double harga;

    public:
        // Constructor
        BarangElektronik(int id, string nama, string jenis, string merek, double harga) {
            this->id = id;
            this->nama = nama;
            this->jenis = jenis;
            this->merek = merek;
            this->harga = harga;
        }

        // Setters
        void setNama(string nama) {
            this->nama = nama;
        }
        void setJenis(string jenis) {
            this->jenis = jenis;
        }
        void setMerek(string merek) {
            this->merek = merek;
        }
        void setHarga(double harga) {
            this->harga = harga;
        }

        // Getters
        int getId() {
            return this->id;
        }
        string getNama() {
            return this->nama;
        }
        string getJenis() {
            return this->jenis;
        }
        string getMerek() {
            return this->merek;
        }
        double getHarga() {
            return this->harga;
        }
};
